api/all/src/main/java/io/opentelemetry/api/trace/TracerBuilder.java

```diff
@@ -28,6 +30,17 @@ public interface TracerBuilder {
    */
   TracerBuilder setInstrumentationVersion(String instrumentationScopeVersion);
 
+  /**
+   * Assigns instrumentation scope attributes for the resulting Tracer.
```

Comment:
I think we should add some context to this documentation. Something like "These attributes will be added to every span created by this Tracer."